semantic_release/vcs_helpers.py

```diff
@@ -1,21 +1,27 @@
 import re
 
-from git import GitCommandError, NoSuchPathError, Repo, TagObject
+from git import GitCommandError, InvalidGitRepositoryError, Repo, TagObject
 
 from .errors import GitError
 from .settings import config
 
 try:
-    repo = Repo('.git', search_parent_directories=True)
-except NoSuchPathError:
+    repo = Repo('.', search_parent_directories=True)
+except InvalidGitRepositoryError:
     repo = None
 
 
+def _check_repo():
```

Comment:
Why does this function name start with an `_`?

Reply:
It is intended to be a private function, not parte of the public API
https://docs.python.org/2/tutorial/classes.html#private-variables-and-class-local-references

For private functions no docstring is required...

Reply:
Please remove the `_`.

Reply:
As you wish... done!